src/components/MoreInformation.js

```diff
@@ -5,60 +5,119 @@ import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
 import { makeStyles } from "@material-ui/core";
 
 const useStyles = makeStyles((theme) => ({
-    accordion: {
-        "margin-top": theme.spacing(2),
-        "margin-bottom": theme.spacing(2),
-        ":last-child": { "margin-bottom": theme.spacing(2) },
-    },
-    buttonText: {
-        ...theme.typography.button,
-    },
-    extraDataContainer: {
-        display: "block",
-    },
-    extraData: {
-        display: "block",
-        "padding-bottom": theme.spacing(1),
-    },
+  accordion: {
+    "margin-top": theme.spacing(2),
+    "margin-bottom": theme.spacing(2),
+    ":last-child": { "margin-bottom": theme.spacing(2) },
+  },
+  buttonText: {
+    ...theme.typography.button,
+  },
+  extraDataContainer: {
+    display: "block",
+  },
+  extraData: {
+    display: "block",
+    "padding-bottom": theme.spacing(1),
+  },
+  formattedExtraData: {
+    color: "#F50057",
+  },
 }));
 
 export default function MoreInformation({ entry }) {
-    const classes = useStyles();
-    return (
-        <Accordion className={classes.accordion}>
-            <AccordionSummary
-                expandIcon={<ExpandMoreIcon />}
-                className={classes.buttonText}
-            >
-                More Information
-            </AccordionSummary>
-            <AccordionDetails className={classes.extraDataContainer}>
-                <div className={classes.extraData}>
-                    <b>Address:</b> {entry.streetAddress}, {entry.city}, MA{" "}
-                    {entry.zip}
-                </div>
-                <ExtraData data={entry.extraData} />
-            </AccordionDetails>
-        </Accordion>
-    );
+  const classes = useStyles();
+  if (!(entry.extraData == null)) {
+    //reggie lewis uses markdown but not for emphasis so it skips that site
+    if (!(entry.location == "Reggie Lewis State Track Athletic Ctr")) {
+      var currentData = entry.extraData["Additional Information"];
+      if (!(currentData == null)) {
+        var splitDataEmStr = currentData.split("***");
+        var i = 1;
+        var newDataEmStr = splitDataEmStr[0];
+        var isEmStr = false;
+        while (i < splitDataEmStr.length) {
```

Comment:
I'd change this to a `for` loop since your variable `i` always increments and has a discrete end.